docs/contentSharing.md

```diff
@@ -76,3 +76,8 @@ Just as anyone with access to a question in your course can access any file in `
 3. Add `"sharePublicly": true` to the `info.json` file of questions you would like to share publicly.
 4. Optionally, add `"shareSourcePublicly": true` to the `info.json` file if you would like people to also be able to view and copy the source code of your question.
 5. Anyone with a PrairieLearn account may preview your question, and any PrairieLearn course may now use it by referencing it in their assessments.
+
+## Steps to share an assessment
```

Comment:
Consider adding H2 headings like "Sharing questions" and "Sharing assessments", moving most of the above under "Sharing questions", adding a brief overview of what assessment sharing is and how it works under "Sharing assessments", and shifting these steps under an H3 heading.

Reply:
ya, this is a good idea. I think I am just going to rearrange this whole docs page right now with the headings you suggested. I will also do a 'Using shared questions' heading, which will be at the top, since I think that will be the most common use.

Reply:
done in f3cc854d8c